ctapipe/ml/tools/tests/test_train.py

```diff
@@ -0,0 +1,28 @@
+from ctapipe.core import run_tool
+from ctapipe.ml import TrainEnergyRegressor
+
+try:
+    from importlib.resources import files
+except ImportError:
+    from importlib_resources import files
+
+
+def resource_file(filename):
```

Comment:
This is the same as in the test for process.
Maybe we can collect these in `conftest.py`?

Reply:
Utils, it's not just for tests